projects/packages/transport-helper/changelog/backup-helper-script-manager-class-versioning

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
This changes the fully-namespaced names of the classes, which is a breaking change.
```suggestion
Significance: major
```